merge/schema_change_test.go

```diff
@@ -231,9 +221,17 @@ func TestAtomicToGranularSchemaChanges(t *testing.T) {
 			`,
 			APIVersion: "v1",
 			Managed: fieldpath.ManagedFields{
+				// Note that manager one previously owned
+				// the top level _P("struct")
+				// which included all of its subfields
+				// when the struct field was atomic.
+				//
+				// Upon changing the schema of struct from
+				// atomic to granular, manager one continues
+				// to own the same fielset as before,
+				// but does not retain ownership of any of the subfields.
```

Comment:
Also make it clear that this is a known limitation and not the ideal beahvior?

Reply:
added